client/flutter/lib/components/page_scaffold/page_scaffold.dart

```diff
@@ -25,7 +25,7 @@ class PageScaffold extends StatelessWidget {
     this.header,
     this.showHeader = true,
     this.padding = EdgeInsets.zero,
-    this.color = CupertinoColors.white,
+    this.color = CupertinoColors.systemGrey6,
```

Comment:
Looks like this grey is slightly different: `#F9F8F7`. Can you define a new constant for this and use that instead?